app/views/configure/shortcut.phtml

```diff
@@ -184,7 +184,7 @@
 		<div class="form-group form-actions">
 			<div class="group-controls">
 				<button type="submit" class="btn btn-important"><?= _t('gen.action.submit') ?></button>
-				<button type="submit" class="btn btn-important" name="load_default_shortcuts"><?= _t('gen.action.load_default_shortcuts') ?></button>
+				<button type="submit" class="btn btn-important confirm" name="load_default_shortcuts"><?= _t('gen.action.load_default_shortcuts') ?></button>
```

Comment:
```suggestion
				<button type="submit" class="btn btn-important confirm" name="load_default_shortcuts" value="1"><?= _t('gen.action.load_default_shortcuts') ?></button>
```

I think it is safer/nicer with some kind of explicit value, as this parameter will be used

Reply:
done!